source/guide_code-server.rst

```diff
@@ -0,0 +1,119 @@
+.. author:: tobimori <tobias@moeritz.cc>
+
+.. tag:: lang-nodejs
+.. tag:: web
+.. tag:: ide
+.. tag:: collaborative-editing 
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/code-server.png
+      :align: center
+
+#############
+code-server
+#############
+
+.. tag_list::
+
+code-server_  is `VS Code`_ running on a remote server, in this guide your Uberspace, accessible through the browser. `VS Code`_ is a modern code editor with integrated Git support, a code debugger, smart autocompletion, and customizable and extensible features. This means that you can use various devices running different operating systems, and always have a consistent development environment on hand.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+code-server_ is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+Setup your Domain:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+=============
+
+Create a new folder for code-server_ in your home directory and switch into it.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mkdir ~/code-server && cd ~/code-server
+  [isabell@stardust code-server]$ 
+
+
+Then download the latest release_ for x86_64. Make sure to the ``x.x.x`` with the current version in all following snippets.
+
+.. code-block:: console
+
+  [isabell@stardust code-server]$ wget https://github.com/cdr/code-server/releases/download/x.x.x/code-server-x.x.x-linux-x86_64.tar.gz
+  [...]
+  ‘code-server-x.x.x-linux-x86_64.tar.gz’ saved
+  [isabell@stardust code-server]$
+
+
+Then extract the files in the current folder with ``tar``. Don't forget ``--strip-components=1`` to remove the ``code-server-x.x.x-linux-x86_64`` prefix from the path.
+
+.. code-block:: console
+
+  [isabell@stardust code-server]$ tar -xzf code-server-x.x.x-linux-x86_64.tar.gz --strip-components=1
+  [isabell@stardust code-server]$
+
+You can now delete the archive:
+
+.. code-block:: console
+
+  [isabell@stardust code-server]$ rm code-server-x.x.x-linux-x86_64.tar.gz
+  [isabell@stardust code-server]$
+
+Setup daemon
+====================
+
+.. note::
+
+    Be aware that almost all configuration of code-server_ happens via command line arguments and so in this file. You can see all available command line arguments when running code-server_ with the ``--help`` argument. For example, you can disable telemetry by adding ``--disable-telemetry``.
+
+Create ``~/etc/services.d/code-server.ini`` with the following content. 
+Make sure to `<password>` with your password.
+
+.. code-block:: ini
+  :emphasize-lines: 3
+
+  [program:code-server]
+  command=%(ENV_HOME)s/code-server/code-server --host 0.0.0.0 --port 8080 --user-data-dir %(ENV_HOME)s/code-server
```

Comment:
`%(ENV_HOME)s/code-server` is the same directory the source of the server itself lives in. Please choose something else like `%(ENV_HOME)s/code-server-data` so they don't get mixed up when users try to update or clean up later.